package.json

```diff
@@ -2,12 +2,14 @@
     "name": "promptifyjs",
     "version": "0.0.1",
     "description": "Prompt Engineering, Solve NLP Problems with LLM's & Easily generate different NLP Task prompts for popular generative models like GPT, PaLM, and more with Promptify",
-    "main": "index.js",
```

Comment:
There is no `dist/index.js` file in the build output. We should create an `index.ts` file inside `src` first.